presto-main/src/test/java/io/prestosql/operator/scalar/TestStringFunctions.java

```diff
@@ -290,6 +290,45 @@ public void testStringPosition()
         testStrPosAndPosition(null, "", null);
         testStrPosAndPosition("", null, null);
         testStrPosAndPosition(null, null, null);
+
+        assertFunction("STRPOS(NULL, '')", BIGINT, null);
+        assertFunction("STRPOS('', NULL)", BIGINT, null);
+        assertFunction("STRPOS(NULL, NULL)", BIGINT, null);
+        assertInvalidFunction("STRPOS('abc/xyz/foo/bar', '/', 0)", "'instance' must be a positive or negative number.");
+        assertInvalidFunction("STRPOS('', '', 0)", "'instance' must be a positive or negative number.");
+
+        assertFunction("STRPOS('abc/xyz/foo/bar', '/')", BIGINT, 4L);
+        assertFunction("STRPOS('\u4FE1\u5FF5,\u7231,\u5E0C\u671B', '\u7231')", BIGINT, 4L);
+        assertFunction("STRPOS('\u4FE1\u5FF5,\u7231,\u5E0C\u671B', '\u5E0C\u671B')", BIGINT, 6L);
+        assertFunction("STRPOS('\u4FE1\u5FF5,\u7231,\u5E0C\u671B', 'nice')", BIGINT, 0L);
```

Comment:
Can you add tests for searching from the back for strings containing non-ASCII unicode characters?